GameData/RealismOverhaul/Engine_Configs/GCRC_Config.cfg

```diff
@@ -0,0 +1,106 @@
+//GCRC (Altair predecessor)
+//Squad
+//
+//
+// 1974072500-The-Vanguard-Satellite-Launching-Vehicle-an-Engineering-Summary.pdf
+// dimensions p.63 & p.65
+// GCRC 33KS-2800 18.25 x 57.38 "  463 x 1457 mm   
+// ABL X-248	  18,00 x 58,2 "   457 x 1483 mm
+// performance, mass, burn time pp. 151
+// GCRC  47lb "inert mass", 383lb propellant, 3lb unspent propellant ISP 235s
+// ABL   50lb "inert mass", 455lb propellant, 8lb liner expelled     ISP 251s 
+// 
+// given dV figures @ ISP only work out if end mass is "inert" mass +3lb or -8lb, respectively
+// hence drymass + propellant
+// 50 + 420 lb  -- 23 + 190 kg  109.1 l
+// 42 + 463 lb  -- 19 + 210 kg  120.65l
+
+
+@PART[*]:HAS[#engineType[GCRC]]:FOR[RealismOverhaulEngines]
+
+{
+	@title = GCRC
```

Comment:
It would be better if these editor fields used the % modifier. It may sound weird and stupid but i have found a lot of mod parts that do not include these in their part configs.